src/Avalonia.Animation/IAnimation.cs

```diff
@@ -11,11 +11,11 @@ public interface IAnimation
         /// <summary>
         /// Apply the animation to the specified control
         /// </summary>
-        IDisposable Apply(Animatable control, IObservable<bool> match, Action onComplete = null);
+        IDisposable Apply(Animatable control, IClock clock, IObservable<bool> match, Action onComplete = null);
 
         /// <summary>
         /// Run the animation to the specified control
         /// </summary>
-        Task RunAsync(Animatable control);
+        Task RunAsync(Animatable control, IClock clock);
```

Comment:
It's not so clear from the API what the difference is between `Apply` vs `RunAsync` and the naming is a bit strange: what's the difference between "Applying" and "Running" an animation? Does `Apply` run the animation _synchronously_? :P

Might be worth getting some API docs to clear up what the usage is for each of these, and maybe trying to unify the naming?

Reply:
I'll try to add in some docs here.